Tests/SwiftDocCUtilitiesTests/PreviewServer/PreviewServerTests.swift

```diff
@@ -201,7 +201,7 @@ class PreviewServerTests {
     }
     
     func testPreviewServerBindDescription() {
-        let localhostBind = PreviewServer.Bind.localhost(port: 1234)
+        let localhostBind = PreviewServer.Bind.localhost(host: "localhost", port: 1234)
```

Comment:
Should we also test that by default, "localhost" is the host?

Reply:
It’s the default in the sense that the `--host` option (in `PreviewOptions`) now has the value `localhost` by default, but `PreviewServer` itself has no default host — an explicit value is always required. Are you suggesting we add a `PreviewOptions` test to check that `--host` gets the right default, or something different? There’s not currently a `PreviewOptionsTests`.

Reply:
Yes, it would be good to test that `--host` gets the right default. The best place for that would be in `PreviewSubcommandTests`, it looks like.